### DISCUSSION TITLE
[BP#96] Ne pas générer de page 404

### DISCUSSION THREAD

**Discussion by Fred-artwai** [cnumr/best-practices#242] [Bonnes pratiques] [answered]
J'avoue le titre m'a fait peur. C'est plus clair dans la description.
Est-ce qu'on pourrait la renommer par "Utiliser une page 404 statique" ? 
voire même des "Utiliser des pages 40X statiques" ?

Car il y aussi les 401, 403 et 410.

**Comment by tbroyer** [comment]
+1

"pages d'erreur" ?

**Comment by ACTLEM** [comment]
+1

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé l'issue pour clarifier le titre (voire la description): https://github.com/cnumr/best-practices/issues/270
